package.json

```diff
@@ -53,7 +55,9 @@
     "instrumentation"
   ],
   "peerDependencies": {
-    "react": "<0.14.0"
+    "react": "<0.15.0 || 0.14.0-rc1",
+    "react-dom": "<0.15.0 || 0.14.0-rc1",
+    "fbjs": "^0.2.0"
```

Comment:
Required because of use of `EventListener` which has been moved out to the `fbjs` package.